src/components/DonationTrackingTable/DonationTrackingTable.jsx

```diff
@@ -0,0 +1,156 @@
+import { useBackend } from '../../contexts/BackendContext';
+import { useState, useEffect } from 'react';
+import DonationSite from './DonationSite';
+import { Tabs, TabList, Tab, Button } from '@chakra-ui/react';
+import { ArrowDownIcon } from '@chakra-ui/icons';
+
+import {
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  TableCaption,
+  TableContainer,
+  Checkbox,
+} from '@chakra-ui/react';
+
+const DonationTrackingTable = () => {
+  const { backend } = useBackend();
+
+  const [donationTrackingTableData, setDonationTrackingTableData] = useState([]);
+  const [checkedSet, setCheckedSet] = useState(new Set());
+  const [topCheckBox, setTopCheckBox] = useState(false);
+
+  const getDonationTrackingTableData = async filter => {
+    const response = await backend.get(`/donation/filter/${filter}`);
+    const data = response.data;
+    const donation_list = [];
+
+    for (let i = 0; i < data.length; ++i) {
+      var donation_site = {
+        donationSite: null,
+        donationId: data[i].donation_id,
+        foodBank: data[i].food_bank_donation,
+        reporter: data[i].reporter,
+        email: data[i].email,
+        date: data[i].date,
+        cannedDogFoodQuantity: data[i].canned_dog_food_quantity,
+        dryDogFoodQuantity: data[i].dry_dog_food_quantity,
+        cannedCatFoodQuantity: data[i].canned_cat_food_quantity,
+        dryCatFoodQuantity: data[i].dry_cat_food_quantity,
+        miscItems: data[i].misc_items,
+        volunteerHours: data[i].volunteer_hours,
+      };
```

Comment:
for future reference, it would be better to only select these columns in the SQL statement to take the load off the frontend. for now you can leave it because we are doing schema changes so we can leave it like this atm